### PR TITLE
Feature/#165 partner flagging verification

### PR DESCRIPTION
**Pull request by elyk**
Summary:
- List/Create endpoints for flags and verifications 
- Detail/Update endpoints for flags and verifications (only allows for is_valid to update)
- Additional serializer for AgencyMember based on the User 

TODO (separate from this PR)
- Tests 
- Addition of AgencyMemberViewer serializer to allow for allow for viewing flags/verifications
- Add in appropriate methods for checking high level status of a partner (flag count, verification status)  on the `Partner` model
- Potentially on above add a `is_verified` on the Partner so we can always store the latest state for easily filtering/ordering

### CODE REVIEW

File: backend/unpp_api/apps/common/consts.py [hunk 1]
```diff
@@ -237,3 +237,8 @@
     ('01m', '_1m', "$100,000,001 to $1,000,000,000"),
     ('Mor', 'more', "More than $1,000,000,000"),
 )
+
+FLAG_TYPES = Choices(
+    ('YLW', 'yellow', 'Yellow Flag'),
```
Comment (lkolacz) [inline backend/unpp_api/apps/common/consts.py]: we use 3 uppercase when it's shortcut of some long name like United State of America -> 'USA' if we have one word like 'Poland' then we use 'Pol'... so in this case we should have `Yel` (or `Ylw`) and `Red`.

File: backend/unpp_api/apps/review/admin.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.contrib import admin
+
+# Register your models here.
```
Comment (lkolacz) [inline backend/unpp_api/apps/review/admin.py]: You should register new models `PartnerFlag, PartnerVerification`.

File: backend/unpp_api/apps/review/models.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    Verification on a Partner
+    """
+    partner = models.ForeignKey(
+        'partner.Partner', related_name="verifications")
+    is_valid = models.BooleanField(default=True)
+    is_verified = models.BooleanField()
+    submitter = models.ForeignKey(
+        'account.User', related_name="given_verifications")
+    is_cert_uploaded = models.BooleanField()
+    cert_uploaded_comment = models.TextField(null=True, blank=True)
```
Comment (lkolacz) [inline backend/unpp_api/apps/review/models.py]: usually comments in this project have max length to 200 chars and in few other place 400 .. we have also few place where text is correct .. but are You sure about it in this case? That we want to have TextFields here?

File: backend/unpp_api/apps/review/serializers.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+class PartnerFlagSerializer(serializers.ModelSerializer):
+
+    submitter = AgencyUserSerializer(read_only=True)
+
+    class Meta:
+        model = PartnerFlag
+        read_only_fields = ('partner', 'submitter',)
+        exclude = ('partner',)
+
```
Comment (lkolacz) [inline backend/unpp_api/apps/review/serializers.py]: 16 and 17 lines number - You are defining 'partner' in read only fields and in exclude .. You should decided what do You want to have here

File: backend/unpp_api/apps/review/serializers.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+class PartnerVerificationSerializer(serializers.ModelSerializer):
+
+    submitter = AgencyUserSerializer(read_only=True)
+
+    class Meta:
+        model = PartnerVerification
+        read_only_fields = ('partner', 'submitter', 'is_verified',)
+        exclude = ('partner',)
```
Comment (lkolacz) [inline backend/unpp_api/apps/review/serializers.py]: as above

File: backend/unpp_api/apps/review/models.py (hunk 3 above)
Comment (elyk) [reply backend/unpp_api/apps/review/models.py]: Yes - they should not be limited.

**Pull request merged**